nucliadb_node/src/services/writer.rs

```diff
@@ -198,14 +197,13 @@ impl ShardWriterService {
         let result = ShardWriterService::initialize(id, path, metadata, tsc, psc, vsc, rsc);
         let metrics = context::get_metrics();
         let took = time.elapsed().map(|i| i.as_secs_f64()).unwrap_or(f64::NAN);
-        let metric = request_time::RequestTimeKey::shard(NAME.to_string());
+        let metric = request_time::RequestTimeKey::shard("writer/new".to_string());
```

Comment:
Dont you thing is better for clarity to use a constant than to have the actual hardcoded value around the code? If is efficiency what your are concern about, the compiler ends up removing replacing constants for the value either way.